src/client/components/Profile/Profile.stories.js

```diff
@@ -1,10 +1,6 @@
 import React from 'react';
 import ProfileDatails from './ProfileDetails';
 
-export default {
```

Comment:
Why did you remove this one here? This is for Storybook.

Reply:
carelessness